src/networkmessage.h

```diff
@@ -44,6 +44,10 @@ class NetworkMessage
 		enum { MAX_PROTOCOL_BODY_LENGTH = MAX_BODY_LENGTH - 10 };
 
 		NetworkMessage() = default;
+		NetworkMessage(const NetworkMessage& other) {
```

Comment:
This doesn't make sense if you don't really plan to make a copy, I see that you modified PR to not make copies, so why this?

Reply:
future proof

Reply:
Although if you ask me, I think that you should allow us to have the freedom of being able to write several events with the same byte, and make a copy of the message for each event that is called, each one decides whether to do more than one event, why put limitations?